test/setup/build.ts

```diff
@@ -21,17 +21,22 @@ async function build() {
   rmSync(tmpTestDir, { recursive: true, force: true });
   mkdirSync(tmpTestDir);
 
-  async function downloadFile(path, destinationPath) {
+  async function downloadFile(path: string, destinationPath: string): Promise<void> {
     const done = promisify(stream.finished);
     const writer = createWriteStream(destinationPath);
 
-    await axios({ method: 'get', url: path, responseType: 'stream' }).then(function (response) {
-      response.data.pipe(writer);
-      return done(writer);
-    });
+    await axios({ method: 'get', url: path, responseType: 'stream' })
+      .then(function (response) {
+        response.data.pipe(writer);
+        return done(writer);
+      })
+      .catch((error) => console.log(error));
```

Comment:
Please only log errors on `stderr` via `console.error`.
Did you actually check what happens here on errors? If I understand correctly you log the error but then continue with the script which is not correct, i.e. if  a step fails you should exit with an error code `!=0`.